src/query/api/v1/handler/prometheus/remote/test/remote/read.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
this hierarchy seems to be a bit too much. Why not just put this within  src/query/test ? Also, maybe have to move write.go from there as well.

Reply:
Was just following the `.../.../write.go` approach, will change